### ISSUE TITLE
Broken link: Agile Methodologies

### ISSUE THREAD

**Issue by twhentschel** [INTERSECT-training/intersect-training.github.io#65]
The link to the slide "Agile Methodologies" by James M. Willenbring doesn't seem to be working for me. I think I found the correct link for the slides though: [https://wordpress.cels.anl.gov/atpesc/wp-content/uploads/sites/96/2020/08/ATPESC-2020-Track-7-Talk-3-Willenbring-AgileProjMgmt-1.pdf](https://wordpress.cels.anl.gov/atpesc/wp-content/uploads/sites/96/2020/08/ATPESC-2020-Track-7-Talk-3-Willenbring-AgileProjMgmt-1.pdf). Found on the [Argonne training program on extreme-scale computing (ATPESC) agenda 2020
](https://extremecomputingtraining.anl.gov/agenda-2020/).

Should I open a PR for this change?

**Comment by JeffCarver** [comment]
Yes, please go ahead and open a PR if you have a link to update. Thanks!
